### PR TITLE
HAI-1510 Implement view for managing project users access rights

### PR DESCRIPTION
**Pull request by markohaarni**
# Description

Implemented view where the user can see all contacts that have been added to project and its applications and manage their access rights.

Users in the table can be sorted and filtered.

Functionality to manage the rights is implemented later.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Other

# Instructions for testing

1. Navigate to hanke that has applications that have been sent to Allu.
2. Click "Hallitse käyttöoikeuksia" button to navigate to access rights view.
3. Check that there are users listed in the table and that they can be sorted by name and email and that the search input works. 

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

# Other relevant info

Please describe here if there is e.g. some requirements for this change or
other info that the tester/user needs to know.

### CODE REVIEW

**Comment by corvidian** [comment]
Opening the user level dropdown breaks the layout, the dropdown opens inside the list block, not over the rest of the page:
image

The dropdown isn't supposed to do anything, so it might be better to disable it, or replace it with just text and re-add it in HAI-1885.

**Comment by markohaarni** [comment]
> Opening the user level dropdown breaks the layout, the dropdown opens inside the list block, not over the rest of the page:
> 
> image
> 
> The dropdown isn't supposed to do anything, so it might be better to disable it, or replace it with just text and re-add it in HAI-1885.

Replaced the dropdown with text until HAI-1885.

File: src/domain/common/components/hankeInformationView/HankeInformationView.tsx [hunk 1]
```diff
@@ -41,10 +41,12 @@ function InformationViewContentContainer({
 
 function InformationViewMainContent({
   children,
+  className,
 }: {
   children: React.ReactNode | React.ReactNode[];
+  className?: string;
 }) {
-  return <div className={clsx(styles.mainContent, styles.paddingLeft)}>{children}</div>;
+  return <div className={clsx(styles.mainContent, styles.paddingLeft, className)}>{children}</div>;
```
Comment (corvidian) [inline src/domain/common/components/hankeInformationView/HankeInformationView.tsx:49]: The only value ever given to className is `style.mainContent`, already mentioned here. I don't really understand the context, but is the className parameter really needed?

Comment (markohaarni) [reply src/domain/common/components/hankeInformationView/HankeInformationView.tsx:49]: Yes, the className parameter is the extra css classes that can be given to that component and it's needed here.

**Review by corvidian — APPROVED** [review]

**Pull request merged**